Dockerfile

```diff
@@ -16,9 +16,12 @@ RUN pnpm fetch
 # copy the rest of the content
 COPY --chown=node:node . /home/node/app
 
+# Prevent pre-commit from being installed, we don't need it here and it fails to install anyway
+ENV SKIP_PRE_COMMIT=true
+
 # `--ignore-scripts` prevents the `prepare` script from running. This avoids
 # installation of pre-commit inside the container.
```

Comment:
The comment can be deleted.
```suggestion
```

Reply:
Whoops, thanks, Krystle! Commited in https://github.com/WordPress/openverse-frontend/pull/1905/commits/5a7334ff1a219de104dd0f7d594990c5ec96d8cd